tests/test_run_rvic.py

```diff
@@ -57,27 +56,33 @@ def full_rvic_test(kwargs, client, valid_input=True):
                 "grid_id": "COLUMBIA",
                 "run_startdate": "2012-12-01-00",
                 "stop_date": "2012-12-31",
-                "pour_points": resource_filename(
-                    "tests", "data/samples/sample_pour.txt"
-                ),
-                "uh_box": resource_filename("tests", "data/samples/uhbox.csv"),
-                "routing": resource_filename(
-                    "tests", "data/samples/sample_flow_parameters.nc"
-                ),
-                "domain": resource_filename(
-                    "tests", "data/samples/sample_routing_domain.nc"
-                ),
-                "input_forcings": url_path(
-                    "columbia_vicset2.nc", "opendap", "climate_explorer_data_prep"
-                ),
-                "params_config_file": resource_filename(
-                    "tests", "data/configs/parameters.cfg"
-                ),
-                "params_config_dict": None,
-                "convolve_config_file": resource_filename(
-                    "tests", "data/configs/convolve.cfg"
-                ),
-                "convolve_config_dict": None,
+                "lons": "-116.46875",
+                "lats": "50.90625",
+                "names": "BCHSP",
+                "long_names": "Spillimacheen",
+            }
+        ),
+        (
+            {
+                "case_id": "sample",
+                "grid_id": "PEACE",
+                "run_startdate": "2012-12-01-00",
+                "stop_date": "2012-12-31",
+                "lons": "-124.90625",
+                "lats": "57.21875",
+                "names": "ARNT7",
+            }
+        ),
+        (
+            {
+                "case_id": "sample",
+                "grid_id": "FRASER",
+                "run_startdate": "2012-12-01-00",
+                "stop_date": "2012-12-31",
+                "lons": "-119.65625",
+                "lats": "50.96875",
+                "names": "ADAMS",
+                "long_names": "ADAMS RIVER NEAR SQUILAX",
```

Comment:
Do you have any test cases that actually use multiple pour points? If not, that would be an important use case to test since you have a fair bit of code to support that.

Reply:
Yeah, I'm planning to test with multiple pour points once I incorporate the new version of `rvic-daccs`.